projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -159,9 +160,11 @@ public function initial_state() {
 				'apiRoot'           => esc_url_raw( rest_url() ),
 				'apiNonce'          => wp_create_nonce( 'wp_rest' ),
 				'registrationNonce' => wp_create_nonce( 'jetpack-registration-nonce' ),
+				'siteSuffix'        => ( new Status() )->get_site_suffix(),
```

Comment:
We can update this when https://github.com/Automattic/jetpack/pull/26135 gets merged.

Reply:
That PR ended up not adding it to the initial state, so we'll probably need to keep that here. I feel like we should be able to combine this state with the editor initial state and use `getSiteFragment` for both, but that's a much bigger change!